src/tests/unit/tests/common/components/__snapshots__/new-tab-link.test.tsx.snap

```diff
@@ -1,27 +1,33 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`NewTabLink handles children 1`] = `
-<StyledLinkBase
-  className="insights-link"
-  href="test"
-  target="_blank"
->
-  link text
-</StyledLinkBase>
+<DocumentFragment>
+  <a
```

Comment:
We should mock the `Link` component for this test.

Reply:
Mocked Link in the file new-tab-link.test.tsx